atc/plan.go

```diff
@@ -137,6 +138,14 @@ type TaskPlan struct {
 	VersionedResourceTypes VersionedResourceTypes `json:"resource_types,omitempty"`
 }
 
+type SetPipelinePlan struct {
+	Name              string `json:"name"`
+	Params            Params `json:"params"`
```

Comment:
Mentioned this already but IMO we shouldn't be using `Params` for this. I would recommend having explicit fields here for `File` and the vars fields.